llvm/test/CodeGen/PowerPC/coalescer-remat-with-undef-implicit-def-operand.mir

```diff
@@ -0,0 +1,84 @@
+# RUN: llc -mtriple=powerpc64le-unknown-linux-gnu -verify-coalescing -run-pass=register-coalescer \
+# RUN:   -o - %s | FileCheck %s
+---
+name:            _Z13testTransposeIfLj31ELj17EEvv
+alignment:       16
+exposesReturnsTwice: false
```

Comment:
Can drop most of these fields (running with -simplify-mir -run-pass=none will also do that)